drupal/Drupal.py

```diff
@@ -290,15 +294,19 @@ def drush_updatedb(repo, branch, build, drupal_version):
 @task
 @roles('app_primary')
 def drush_fra(repo, branch, build, drupal_version):
-  print "===> Reverting all features..."
-  with settings(warn_only=True):
-    if sudo("su -s /bin/bash www-data -c 'cd /var/www/%s_%s_%s/www/sites/default && drush -y fra --force'" % (repo, branch, build)).failed:
-      print "Could not revert features! Reverting database and settings..."
-      Revert._revert_db(repo, branch, build)
-      Revert._revert_settings(repo, branch, build)
-      raise SystemExit("Could not revert features! Site remains on previous build")
+  with cd("/var/www/%s_%s_%s/www/sites/default" % (repo, branch, build)):
+    if run("drush pm-list --pipe --type=module --status=enabled --no-core | grep -q ^features$"):
+      print "===> Features module not installed, skipping feature revert"
     else:
-      drush_clear_cache(repo, branch, build, drupal_version)
+      print "===> Reverting all features..."
+      with settings(warn_only=True):
+        if sudo("su -s /bin/bash www-data -c 'cd /var/www/%s_%s_%s/www/sites/default && drush -y fra'" % (repo, branch, build)).failed:
```

Comment:
Because there's now a `with cd()` on line 297, I don't think we need to cd into /var/www/%s_%s_%s/www/sites/default here?

Reply:
Good spot, I've removed it in the latest commit